.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Test and Build React
+
+on:
+    push:
+
+    pull_request:
+        branches: ['master']
```

Comment:
Can you change this to be on both the `master` and `main` branches? We currently are using `main` as the primary, but would be nice to cover both

Reply:
Oops my bad

Reply:
Leftover from the project I copied this from